client/layout/sidebar/style.scss

```diff
@@ -199,6 +199,7 @@
 .sidebar__menu.is-togglable .sidebar__menu-link {
 	line-height: 1.5;
 	padding: 7px 16px 7px 20px;
+	padding-left: 55px;
 }
```

Comment:
IMO it'd be slightly better to move these into the `sidebar__menu.is-togglable` section below, just to keep all those styles in the same place

Reply:
I agree but I wanted to only fix the lining errors and leave further improvements for future PRs.